CHANGELOG.rst

```diff
@@ -1,3 +1,48 @@
+Version build - March 04, 2018
```

Comment:
I think there is no need to add the changelog here, but looks great how is generated now :)

Reply:
Thank you... But hope I didn't messed up anything in my first attempt :P

Reply:
It's ok, just revert the version number, delete the changelog file and take a look to my other comment. Then I think it's ready to get merged! Thanks for you interest on collaborating.